### ISSUE TITLE
[android] Pokemon Lets go Eevee bluetooth joystick problem

### ISSUE THREAD

**Issue by Rhouelyn** [yuzu-emu/yuzu#12620]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

v186

### Description of Issue

Yuzu v186
Snapdragon 870
6gb ram
Game: Pokemon Lets go Eevee
Issue: Player cant properly run when using bluetooth controller.
My Controller: Gulikit NS09 (hall effect sticks.)
*Other games walk and run fine using my controller, only Pokemon Lets go have this problem.

https://github.com/yuzu-emu/yuzu/assets/78578824/17aad1d5-776b-401e-8b37-e9a6af1515d5

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13859852/yuzu_log.txt.old.txt)

### Expected Behavior

Yuzu v186
Snapdragon 870
6gb ram
Game: Pokemon Lets go Eevee
Issue: Player cant properly run when using bluetooth controller.
My Controller: Gulikit NS09 (hall effect sticks.)
*Other games walk and run fine using my controller, only Pokemon Lets go have this problem.

https://github.com/yuzu-emu/yuzu/assets/78578824/77a17832-abab-4fc2-8fa0-08737675451a

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13859861/yuzu_log.txt.old.txt)

### Reproduction Steps

Yuzu v186
Snapdragon 870
6gb ram
Game: Pokemon Lets go Eevee
Issue: Player cant properly run when using bluetooth controller.
My Controller: Gulikit NS09 (hall effect sticks.)
*Other games walk and run fine using my controller, only Pokemon Lets go have this problem.

### Log File

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13859866/yuzu_log.txt.old.txt)

### System Configuration

CPU: Snapdragon 870
GPU/Driver: turnip r12 agressive vulkan beta
RAM: 6gb ram
OS: Zui 14

**Comment by jasmohan-narula** [comment]
Might be an issue with the controller. I had something similar happen with mine. I opened it up, and sprayed some WD40 inside the joysticks which improved the useability but did not entirely fix it.

**Comment by Rhouelyn** [comment]
No, my controller works fine, like i said this only happens on pokemon lets go, and look at the end of the video you will see me testing the stick with no issues perfect circle.
